LambdaS3FileZipper/Aws/AwsS3Client.cs

```diff
@@ -14,12 +13,9 @@ public class AwsS3Client : IAwsS3Client
 	{
 		private readonly IAmazonS3 client;
 
-		public RegionEndpoint RegionEndpoint { get; }
-
-		public AwsS3Client(RegionEndpoint regionEndpoint)
+		public AwsS3Client(IAmazonS3 client)
```

Comment:
Should we create a default value for `IAmazonS3`?

Reply:
I'll do that as part of another pull request.